pkgs/applications/office/clickup/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib, stdenv, fetchurl, appimageTools, makeWrapper, undmg, unzip }:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  pname = "clickup";
+  name = "clickup-desktop";
+  appname = "ClickUp";
+  version = "2.0.23";
```

Comment:
Seems outdated according to clickup/clickup-release#6? https://clickup.com/download is already at 3.1.2 and has aarch64-darwin support.

Reply:
@wegank You're right, that the GitHub releases are outdated, but I verified that they still work. It seems as if ClickUp updates itself upon first launch anyways.
Should I just target the download page for the sources instead?

Reply:
The sources are at

- `https://download.todesktop.com/210531zdwwjv8ke/ClickUp 3.1.2-arm64.dmg` for aarch64-darwin,
- `https://download.todesktop.com/210531zdwwjv8ke/ClickUp 3.1.2-x64.dmg` for x86_64-darwin,
- `https://download.todesktop.com/210531zdwwjv8ke/ClickUp-3.1.2.AppImage` for x86_64-linux.

Reply:
I updated the PR to target the official download links as sources. This way I was able to also add aupport for aarch64-darwin. PR is ready for review.